### PR TITLE
[2.3.32] patch to longstanding bug with EEPROM JSON

### PR DESCRIPTION
**Pull request by trestoh**

### CODE REVIEW

File: WasatchNET/EEPROMJSON.cs [hunk 1]
```diff
@@ -334,14 +334,14 @@ public string ToString(int level, int indentSize = 2)
...
+                addField(sb, indent, "RelIntCorrCoeffs", RelIntCorrCoeffs);
             
             addField(sb, indent,"Bin2x2", Bin2x2);
-            sb.AppendFormat("{0}\"{1}\": {2}", indent, "FlipXAxis", FlipXAxis ? "true" : "false");
+            addField(sb, indent, "FlipXAxis", FlipXAxis);
             addField(sb, indent, "Gen15", Gen15);
             addField(sb, indent, "CutoffFilter", CutoffFilter);
             addField(sb, indent, "EvenOddHardwareCorrected", EvenOddHardwareCorrected);
-            addField(sb, indent, "LaserWarmupS", LaserWarmupS);
+            sb.AppendFormat("{0}\"{1}\": {2}", indent, "LaserWarmupS", LaserWarmupS);
```
Comment (mzieg) [inline WasatchNET/EEPROMJSON.cs:344]: Out of curiosity, why this change?

**Review by mzieg — APPROVED** [review]

File: WasatchNET/EEPROMJSON.cs (hunk 1 above)
Comment (trestoh) [reply WasatchNET/EEPROMJSON.cs:344]: our upstream json writer _really_ doesn't like floating commas for how it appends fields together and having an extra comma after this field causes bad writes that end up confusing our parsers
